WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Notes Section/Customer Note/EditCustomerNote.swift

```diff
@@ -120,6 +121,16 @@ struct EditCustomerNote<ViewModel: EditCustomerNoteViewModelProtocol>: View {
     }
 }
 
+// MARK: View extensions
+private extension View {
+
+    /// Set autofocus to TextEditor
+    ///
+    func focused() -> some View {
+        self.modifier(AutofocusTextField())
+    }
+}
```

Comment:
Could you move this to the `AutofocusTextField` file as a public extension so it can be used by everyone?
Also, lets make sure to add proper documentation so consumers know that it only works on iOS15+

Reply:
I will make these changes.